Engine/csound_orc_compile.c

```diff
@@ -1455,15 +1381,16 @@ void insert_opcodes(CSOUND *csound, OPCODINFO *opcodeInfo,
         i = (engineState->maxopcno > 0 ? engineState->maxopcno
              : engineState->maxinsno);
         engineState->maxopcno = i + MAXINSNO;
-        engineState->instrtxtp = (INSTRTXT **)csound->ReAlloc(
-                                                              csound, engineState->instrtxtp,
-                                                              (1 + engineState->maxopcno) * sizeof(INSTRTXT *));
+        engineState->instrtxtp = (INSTRTXT **)
+          csound->ReAlloc(csound, engineState->instrtxtp,
+                          (1 + engineState->maxopcno) * sizeof(INSTRTXT *));
         /* Array expected to be nulled so.... */
         while (++i <= engineState->maxopcno)
           engineState->instrtxtp[i] = NULL;
       }
       inm->instno = num;
-      // csound->Message(csound, Str("UDO INSTR NUM: %d\n"), num);
+      if(csound->GetDebug(csound))
```

Comment:
Related to earlier comment, we might find it better to introduce a csound->Log(csound, log_level, formatted string, args...) method that introduces different log levels like info, warn, error, etc. This is pretty common and would hide things like checking GetDebug() and simplify the codebase IMO.